build.ps1

```diff
@@ -51,8 +51,9 @@ if($Verbose.IsPresent)
 
 $TOOLS_DIR = Join-Path $PSScriptRoot "tools"
 $NUGET_EXE = Join-Path $TOOLS_DIR "nuget.exe"
-$CAKE_EXE = Join-Path $TOOLS_DIR "Cake/Cake.exe"
+$CAKE_EXE = "dotnet dotnet-cake"
```

Comment:
This is more of a command, than an exe. I'd refactor and just call `dotnet dotnet-cake tool ...` inline below.